phabfive/maniphest.py

```diff
@@ -14,14 +17,83 @@
 import yaml
 from jinja2 import Template
 
-
 log = logging.getLogger(__name__)
 
-
 class Maniphest(Phabfive):
     def __init__(self):
         super(Maniphest, self).__init__()
 
+    def alex_search(self, created_after=None, updated_after=None, project=None):
```

Comment:
Would you mind renaming the function to something more suitable, i.e. that doesn't include your name?